src/modules/master_transforms/include/master_tf.h

```diff
@@ -0,0 +1,73 @@
+#ifndef MASTER_TF_H
+#define MASTER_TF_H
+
+/*
+NOTE: This node prepares the tf message to broadcast by using the input of the rotated yaw message
+      (according to REP-105) and performing the 2D rotation required from sensor location to front-axle. 
+      Only the x-position is offset (with respect to the vehicle on the red car) so there is no y-position component in the 
+      2-D transform. 
+
+INPUTS: TOPIC:  /localization/rotated_yaw
+                Msg: std_msgs::Float32
+        TOPIC:  /localization/calibrated_yaw
+                Msg: std_msgs::Float32
+
+OUTPUTS: Geodesy TF Broadcast
+         Lidar TF Broadcast
+*/
+
+#include "ros/ros.h"
+#include "math.h"
+#include "nav_msgs/Odometry.h"
+#include "std_msgs/Float32.h"
+#include "std_msgs/Bool.h"
+#include "tf2_ros/transform_broadcaster.h"
```

Comment:
Choose either ```tf2``` or ```tf``` for the header selection. I recommend tf2 because ```tf``` is deprecated.

Reply:
I'm going to choose `tf` because it is a lot simpler and more straight forward than `tf2`.

Reply:
Sounds good.